docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -100,31 +143,68 @@ The database settings screen will give you a number of options for your applicat
 
 Once you've entered a password and clicked `Save`, the red outline on the Database box should have gone away, indicating that the application is valid and ready to be launched.
 
-### Additional Options
+#### Configuring RDS for Metabase OUTSIDE ElasticBeanstalk (recommended)
+
+In AWS Console, enter RDS in the search box or select the service from the dropdown button on the top left of the page.
+
+Once inside RDS, click on the `Create database` button
+![RDS Create Database](images/RDSCreateDatabase.png)
+
+On the Create Database menu, you have to select MySQL or PostgreSQL as engine types, as these two are the ones that Metabase support as the Application Database (where Metabase will save all of its configurations). For this example we will choose PostgreSQL on its latest version available in AWS at this time (12.4-R1)
+![RDS Postgres Selection](images/RDSPostgresSelection.png)
+
+On the Templates section, you can leave `Production` selected or choose any other option that better suits your needs.
+![RDS Templates Section](images/RDSTemplatesSection.png)
+
+On the Settings section, you need to type unique identifier for your database and a master password that you will need to preserve for configuring it on the Metabase instances later on this guide.
+![RDS Templates Section](images/RDSPostgresSettings.png)
+
+On the instance size section, you can choose any instance size that will fit your needs. As this is the application database for Metabase, the sizing of the RDS instance depends on the number of Metabase instances that will be connected to this database, number of users that are using Metabase at once and the number of questions, dashboards and configurations that are saved.
+![RDS Instance size](images/RDSInstanceSize.png)
+
+On production deployments, you should be using a Multi-AZ(Availability Zone) cluster, as this will ensure that the database does not goes down in case there is an issue on a single availability zone 
+![RDS MultiAZ](images/RDSMultiAZ.png)
+
+On the Connectivity section, you have to ensure that you are deploying the database in the same VPC as the one you deployed the Metabase instance, otherwise they won't be able to see each other. You have to select to create a VPC security group on the section, as you will need to grant access from Metabase instance to the database on the port that listens for connections.
+![RDS VPC Security Groups](images/RDSVPCSecurityGroup.png)
+
+On the Additional configuration section you have to specify an `Initial database name` as this will be the database that Metabase will use for all of its configurations. From here you can also configure the backup window in case you need to restore the backups at some point in time.
+![RDS Initial Database](images/RDSInitialDatabase.png)
+
+Once this is done, click on the `Create database` button on the lower right part of the page and wait for the database to be created (this can take several minutes).
```

Comment:
Once what is done?

Reply:
changed it to "When you have completed all these configurations", thanks!